Userland/Applications/Debugger/main.cpp

```diff
@@ -47,8 +47,15 @@ static void handle_print_registers(PtraceRegisters const& regs)
     outln("r12={:p} r13={:p} r14={:p} r15={:p}", regs.r12, regs.r13, regs.r14, regs.r15);
     outln("rip={:p} rflags={:p}", regs.rip, regs.rflags);
 #elif ARCH(AARCH64)
-    (void)regs;
-    TODO_AARCH64();
+    outln("x0 ={:p} x1 ={:p} x2 ={:p} x3 ={:p}", regs.x[0], regs.x[1], regs.x[2], regs.x[3]);
+    outln("x4 ={:p} x5 ={:p} x6 ={:p} x7 ={:p}", regs.x[4], regs.x[5], regs.x[6], regs.x[7]);
+    outln("x8 ={:p} x9 ={:p} x10={:p} x11={:p}", regs.x[8], regs.x[9], regs.x[10], regs.x[11]);
+    outln("x12={:p} x13={:p} x14={:p} x15={:p}", regs.x[12], regs.x[13], regs.x[14], regs.x[15]);
+    outln("x16={:p} x17={:p} x18={:p} x19={:p}", regs.x[16], regs.x[17], regs.x[18], regs.x[19]);
+    outln("x20={:p} x21={:p} x22={:p} x23={:p}", regs.x[20], regs.x[21], regs.x[22], regs.x[23]);
+    outln("x24={:p} x25={:p} x26={:p} x27={:p}", regs.x[24], regs.x[25], regs.x[26], regs.x[27]);
+    outln("x28={:p} fp ={:p} lr ={:p} pstate={:p}", regs.x[28], regs.x[29], regs.x[30], regs.spsr_el1);
+    outln("sp ={:p} pc ={:p}", regs.sp, regs.pc);
```

Comment:
I would print the PC, SP, and PSTATE in the same format as here:
https://github.com/SerenityOS/serenity/blob/b7ce8ab808b67da3cf5bcc3ca0ce69a05900d015/Userland/Applications/CrashReporter/main.cpp#L134-L135
(similar to RISC-V a couple of lines down, just also with SP and PSTATE since RISC-V's SP is a GPR and RISC-V doesn't have condition codes)

I'm not sure if you should also call x29 and x30 fp and lr.